install_config/configuring_vsphere.adoc

```diff
@@ -239,6 +256,11 @@ kubeletArguments:
 
 ----
 +
+[NOTE]
```

Comment:
Since this is OKD only, this note should be removed.

Reply:
I'm not sure it has to be OKD only, in fact, I've tested it in OCP 3.9. 
The key point of this PR is to clarify that since kubernetes 1.9, the credentials file is not necessary to be deployed it in nodes (just in masters).